pkg/workloads/cortex/lib/type/api.py

```diff
@@ -35,7 +35,6 @@ def __init__(self, provider, storage, model_dir, cache_dir=".", **kwargs):
         self.key = kwargs["key"]
         self.metadata_root = kwargs["metadata_root"]
         self.name = kwargs["name"]
-        self.endpoint = kwargs["endpoint"]
```

Comment:
This change should be a part of the networking move PR because this code path should break in this PR https://github.com/cortexlabs/cortex/pull/1151